lib/dialects/abstract/index.js

```diff
@@ -7,7 +7,6 @@ AbstractDialect.prototype.supports = {
   'DEFAULT VALUES': false,
   'VALUES ()': false,
   'LIMIT ON UPDATE': false,
-  'ON DUPLICATE KEY': true,
```

Comment:
Why?

Reply:
No longer used, `supports.inserts.updateOnDuplicate` already has this information